.github/workflows/ci_push_testing.yml

```diff
@@ -0,0 +1,239 @@
+      run: python3 -m pytest --cov-report term-missing --cov ganga/GangaDirac ganga/GangaDirac/test/GPI
+
+  lhcb-unit:
+    name: GangaLHCb Unit
+    needs: lint
+    runs-on: ubuntu-latest
+    container: 
+      image: centos:7
+      options: --privileged
+    steps:
+    - name: Install CVM-FS
+      run: |
+        yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+        yum install -y cvmfs cvmfs-config-default
+        cvmfs_config setup
+        echo "CVMFS_REPOSITORIES=lhcb.cern.ch,sft.cern.ch" > /etc/cvmfs/default.local
+        echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
+        mkdir -p /cvmfs/lhcb.cern.ch
+        mkdir -p /cvmfs/sft.cern.ch
+        mount -t cvmfs lhcb.cern.ch /cvmfs/lhcb.cern.ch
+        mount -t cvmfs sft.cern.ch /cvmfs/sft.cern.ch
+    - uses: actions/checkout@v1
+    - name: Install virtualenv
+      run: |
+        source /cvmfs/lhcb.cern.ch/lib/LbEnv && source LbLogin.sh
+        /cvmfs/sft.cern.ch/lcg/releases/LCG_97python3/Python/3.7.6/x86_64-centos7-gcc9-opt/bin/python3 -m venv ~/venv
+    - name: Install dependencies
+      run: |
+        . ~/venv/bin/activate
+        python3 -m pip install --upgrade pip setuptools wheel
+        python3 -m pip install -e .[LHCb]
+        python3 -m pip install coverage pytest pytest-cov pytest-pylint pytest-mock
+    - name: Test with pytest
+      run: |
+        . ~/venv/bin/activate
+        python3 -m pytest --cov-report term-missing --cov ganga/GangaLHCb ganga/GangaLHCb/test/Unit
+  
+  lhcb-integration:
+    name: GangaLHCb Integration
+    needs: lhcb-unit
+    runs-on: ubuntu-latest
+    container: 
+      image: centos:7
+      options: --privileged
+    steps:
+    - name: Install CVM-FS
+      run: |
+        yum install -y https://ecsft.cern.ch/dist/cvmfs/cvmfs-release/cvmfs-release-latest.noarch.rpm
+        yum install -y cvmfs cvmfs-config-default
+        cvmfs_config setup
+        echo "CVMFS_REPOSITORIES=lhcb.cern.ch,sft.cern.ch" > /etc/cvmfs/default.local
+        echo "CVMFS_HTTP_PROXY=DIRECT" >> /etc/cvmfs/default.local
+        mkdir -p /cvmfs/lhcb.cern.ch
+        mkdir -p /cvmfs/sft.cern.ch
+        mount -t cvmfs lhcb.cern.ch /cvmfs/lhcb.cern.ch
+        mount -t cvmfs sft.cern.ch /cvmfs/sft.cern.ch
+    - uses: actions/checkout@v1
+    - name: Install virtualenv
+      run: /cvmfs/sft.cern.ch/lcg/releases/LCG_97python3/Python/3.7.6/x86_64-centos7-gcc9-opt/bin/python3 -m venv ~/venv
+    - name: Install dependencies
+      run: |
+        . ~/venv/bin/activate
+        python3 -m pip install --upgrade pip setuptools wheel
+        python3 -m pip install -e .[LHCb]
+        python3 -m pip install coverage pytest pytest-cov pytest-pylint pytest-mock
+    - name: Test with pytest
+      env:
+        GANGA_CONFIG_PATH: GangaLHCb/LHCb.ini
+      run: |
+        source /cvmfs/lhcb.cern.ch/lib/LbEnv && source LbLogin.sh
+        . ~/venv/bin/activate
+        lhcb-proxy-init
```

Comment:
I have no idea why the `lhcb-proxy-init` fails. 
```bash
(ganga) [centos7] ~/programming/ganga % which lhcb-proxy-init
/cvmfs/lhcb.cern.ch/lib/var/lib/LbEnv/796/stable/x86_64-centos7/bin/lhcb-proxy-init
```
can you take a look at the `PATH`?

Reply:
`PATH` is correctly updated but seems it doesn't matter. Solution was to give absolute path to `lhcb-proxy-init`. I assume this is to do with the fact it's on cvmfs.